### ISSUE TITLE
Move the docs to Antora

### ISSUE THREAD

**Issue by mbroz2** [OpenLiberty/openliberty.io#1243]
Prototype Antora (https://antora.org/ ) for use with our docs.  Under this work, we'd like to just get a branch  with antora building a doc that is displayed on openliberty.io, not actually converting all our existing docs over to antora.  In other words, this could be a separate section of our docs that would be built using Antora.  This would provide the docs team with a sandbox to try the library out without us investing too much into it yet.

Things left to implement:
- [x] Fix mobile navbar when opening the toc drawer
- [x] Go through and check for any bugs like table rendering
- [x] Import Bunuelo fonts into Antora docs
- [x] Get the hash links to jump to the correct spot
- [x] Get the toolbar page menu to appear
- [x] Make sure all of our code styling matches what is found on openliberty.io (We use highlightjs now but it can be customized to be the same)
- [x] Edit config page to have a breadcrumb like we used to
- [x] Remove the page version picker and style the doc version picker like the designs in https://github.com/OpenLiberty/openliberty.io/issues/1526
- [x] Combine the feature versions into combined html and add the version picker to the top of the pages #1526 
- [x] Fix config/feature macros to work with Antora
- [x] Replace the header links with what we have in https://github.com/OpenLiberty/openliberty.io/issues/1273 
- [x] Remove the breadcrumb in docs
- [x] Redirect old hashed pages to the new pages using javascript
- [x] Enable google analytics on the Antora pages *only* on the production site
- [x] Update the head-info.hbs content to have the same seo title and description as we have on our site so when you paste a link to a doc it has the page/site's seo title/description
- [x] Add Operating System tab support like what we have in guides currently.
- [x] Add open an issue link
- [ ] Add page to show versions of Liberty that a topic exists on if the user clicks a version in the version switcher that doesn't have it.
- [x] Update the docs content to what is in master
- [x] Figure out how to link from one doc to the latest version of a doc (including when there's only a single version of a feature doc, in which case the lack of version specified should 'redirect' to that single version).
We used to do this with a script which would go through and change all to point to latest.  This will be provided by Antora (https://gitlab.com/antora/antora/issues/314, similar discussion in https://gitlab.com/antora/antora/issues/291), so I don't think we should duplicate the functionality.

Once Antora is ready, we will 
1. Copy the current `draft` branch content into the `antora-draft` branch making the necessary directory changes for Antora. No more changes can be made to the `draft` branch so we do not lose content transitioning to Antora.
2. Test the draft changes on the Antora staging site.
3. Rename the `antora-draft` branch to `draft`.
4. Changes can be delivered to the `draft` branch again (using Antora pages).
5. Repeat for the `develop` branch using `antora-develop`.
6. Repeat for `master` using the `antora` branch but rename the branch  to `latest`.

**Comment by k8vance88** [comment]
Issue #1393 will not need to be fixed if we go with antora.  Please close that issue when we have a decision.

**Comment by steven1046** [comment]
We need to look into if we can display a flag/link to the latest version of a doc if a user opens an older version of a doc with a newer version available.

**Comment by steven1046** [comment]
We need to update the header for each non latest page to point to the latest version of a page (at least for Javadocs) using the canonical url meta tag in the head.

**Comment by steven1046** [comment]
Per the site discussion, we want to remove the "page" picker and update the global docs version picker in the left TOC to take you to take you to the same page you were on in the version you select **if it exists**, otherwise just take you to the new version's landing page.

**Comment by mdenby** [comment]
Here are redlines for the Docs and Guides TOC. The rollover and select states should work how they are currently working on the Docs page.

[Archive.zip](https://github.com/OpenLiberty/openliberty.io/files/4457751/Archive.zip)
<img width="412" alt="Screen Shot 2020-04-09 at 1 37 07 PM" src="https://user-images.githubusercontent.com/26823018/78924093-3ce3be80-7a67-11ea-87d7-09e31cf35f9f.png">

**Comment by mdenby** [comment]
One question I had was the Java page--should we navigate to a new page like this?

![Docs Java 1](https://user-images.githubusercontent.com/26823018/78924305-8df3b280-7a67-11ea-80d0-47292cd74a60.jpg)

**Comment by steven1046** [comment]
That's a good idea! Let me see if i can get the javadoc's TOC integrated into our own like that because having two is weird.

**Comment by mbroz2** [comment]
Although that looks great, my guess is that unfortunately we won't be able to integrate the javadoc ToC into the docs ToC without a large effort, but I'll differ to @steven1046 regarding actual feasibility & cost.

However, if we moved to generating javadoc with `--no-frames` (@NottyCode ) then there would be no Package/Classes ToC and just a single page which would likely be easier to deal with.  If we could somehow just generate asciidoc (or even markdown) pages instead of html for the javadoc, that would be great since we could then integrate it directly into Antora, however I'm not aware of javadoc having such functionality nor have I looked to see if there's something out there that already does that.
